tests/test_vacuum_importance_sampling.py

```diff
@@ -5,13 +5,14 @@
 import pytest
 
 from timemachine import testsystems
+from timemachine.constants import DEFAULT_FF
 from timemachine.ff import Forcefield
 from timemachine.md import enhanced
 from timemachine.potentials import bonded
 
 
 def get_ff_am1ccc():
-    ff = Forcefield.load_from_file("smirnoff_1_1_0_ccc.py")
+    ff = Forcefield.load_from_file(DEFAULT_FF)
```

Comment:
random idea for quality-of-life improvement: `Forcefield.load_default()`? (not necessary or related to this PR)

Reply:
It would be nice to make forcefield loading in the tests more consistent.  `Forcefield.load_default()` seems like a good approach. Perhaps a fixture as well?